consensus/src/chained_bft/event_processor.rs

```diff
@@ -417,13 +380,7 @@ impl<T: Payload> EventProcessor<T> {
             }
         };
 
-        self.network
-            .broadcast_timeout_msg(TimeoutMsg::new(
-                self.gen_sync_info(),
-                PacemakerTimeout::new(round, self.block_store.signer(), vote_msg_to_attach),
-                self.block_store.signer(),
-            ))
-            .await;
+        self.network.broadcast_vote(timeout_vote_msg).await
```

Comment:
we can probably leave the `execute_and_vote` unchanged and add the round signature above this line